core_types/src/links_entry.rs

```diff
@@ -99,16 +121,142 @@ impl LinkListEntry {
             links: links.to_vec(),
         }
     }
+
+    pub fn links(&self) -> &Vec<Link> {
+        &self.links
+    }
+}
+
+impl ToString for LinkListEntry {
+    fn to_string(&self) -> String {
+        serde_json::to_string(self).expect("LinkEntry failed to serialize")
```

Comment:
Error message should be "LinkListEntry"

Reply:
@ddd-mtl please make edits like this directly when you see them